Drop-In/src/main/java/com/braintreepayments/api/dropin/DropInActivity.java

```diff
@@ -328,6 +328,8 @@ protected void onActivityResult(int requestCode, final int resultCode, Intent da
          * The temporary fix is to forward the data back to BraintreeFragment when we detect
          * the 79129 requestCode.
          */
+        super.onActivityResult(requestCode, resultCode, data);
+
         if (requestCode == 79129) {
```

Comment:
Since we call `super.onActivityResult`, can this work around be dropped? 

See [Edit2](https://github.com/braintree/braintree-android-drop-in/issues/134#issue-461217287)

Reply:
Gotcha! My initial thought was that the workaround was still needed as a backup. But I appreciate you pointing it out again. Reading through that a second time makes sense as to why this workaround can now be removed with the super call added. 👍